### DISCUSSION TITLE
Units of measure for energy logistics

### DISCUSSION THREAD

**Discussion by emoryhunt** [ucum-org/ucum#294] [Q&A]
Hi UCUM community,
 
An application I am working on requires ca. 30 units of measure that are not currently declared in UCUM. Should I submit the entire list as a single request (issue) for consideration as part of the standard, or instead individually?
 
Background: The application is an API for quantity-conversion equations used mostly in the oil/gas/energy logistics industries for custody-transfer operations. A number of units of measure commonly used in this area are not currently declared in UCUM and I've attempted to express these in accordance with the UCUM syntax rules.
 
Many thanks for a hint!

**Comment by timbrisc** [comment]
Hello @emoryhunt, I suggest submitting the requests together en masse so our committee has the context in a single issue. Please do enumerate the full list of units you feel should be referenced in UCUM. 

Thank you for asking.
